### DISCUSSION TITLE
Unable to get emails to send for new accounts?

### DISCUSSION THREAD

**Discussion by caldwellcreativela** [mozilla/hubs#4732] [Hubs Cloud Questions]
Hi all 

I have a hubs space that I got up on Hubs Cloud, but no one can get an email to confirm their accounts? I tried to update the SMTP setting to my GoDaddy server but no luck 

any help would be appreciated
thanks

**Comment by pattersonbl2** [comment]
Hey, did you check to see if your domain is configured correctly in AWS SES?
